src/websocketNotification.ts

```diff
@@ -46,11 +56,33 @@ export declare interface WebsocketNotification {
    */
   // TODO: use more specific type than object in the future
   on(event: "message", listener: (notification: object) => void): this;
+  /**
+   * Emitted when a valid notification is received, the payload is a
+   * [activitystreams
+   * Activity](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-activity).
+   */
```

Comment:
The function name is quite self-documenting, but shouldn't this comment mention that the listener will only be called once? Ditto for the error.